Scripts/3_Deploy.ps1

```diff
@@ -216,6 +216,98 @@ If (!( $isAdmin )) {
         <SkipUserOOBE>true</SkipUserOOBE> 
       </OOBE>
       <TimeZone>$TimeZone</TimeZone>
+      <FirstLogonCommands>
+                <SynchronousCommand wcm:action="add">
+                    <CommandLine>cmd.exe /c powershell -Command "Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Force"</CommandLine>
```

Comment:
Why do we do this? I'm trying to have the environment as vanilla as possible. I'll add this as optional feature. Good idea!

Reply:
Yeah, was not sure how to add it as an option. Not that too advanced yet :)